src/cli.rs

```diff
@@ -324,7 +324,13 @@ pub enum Command {
     /// Configuration profile commands
     Config(command::Config),
 
-    /// Develop a GraphQL API
+    /// Initialize a local supergraph development server.
+    ///
+    /// Run `rover dev` in multiple subgraph project directories,
+    /// and Rover will compose them together and start a dev instance of the Apollo Router. You can work with any GraphQL API
+    /// that you have an SDL file for, or has introspection enabled.
```

Comment:
```suggestion
    /// The first `rover dev` process you run starts a dev instance of the Apollo Router and connects it to the running subgraph you specify.
    /// You can then run additional instances of `rover dev` to add more subgraphs to your local supergraph (the same router instance is used).
    /// As you add subgraphs, `rover dev` automatically composes all subgraph schemas into a new supergraph schema for the router.
    ///
    /// The router instance is tied to the *first* `rover dev` process. If you terminate that process, the router terminates.
```

Reply:
maybe one more thing to add here - if you kill one of the _child_ subgraphs, the main `rover dev` session will _decompose_ it from the supergraph schema

Reply:
Threw another line in there for that